app/templates/test.js

```diff
@@ -1,9 +1,9 @@
 import test from 'ava';
-import fn from '.';
+import m from '.';
```

Comment:
I'm perfectly fine with this, but I think we already discussed this somewhere in an issue and decided not to do it.